doc/base_images.md

```diff
@@ -15,6 +15,11 @@ packages required at build-time. The following system packages are also added
 to the Oryx image. If you require additional packages please [open an
 issue][].
 
+Support various versions of build image:
+[Buster](../images/build/Dockerfiles/githubRunner.BuildPackDepsBuster.Dockerfile),
+[Focal](../images/build/Dockerfiles/githubRunner.BuildPackDepsFocal.Dockerfile),
```

Comment:
Instead of Focal I would say Ubuntu ... and focal is the Ubuntu version so we can probably say Ubuntu (focal)